.eslintrc.js

```diff
@@ -0,0 +1,69 @@
+const { off } = require("process");
```

Comment:
Add to eslint rules: tab indentation = 4 spaces

Reply:
Done